rts/motoko-rts/src/persistence/compatibility.rs

```diff
@@ -26,7 +26,7 @@ const DEFAULT_VALUE: Value = Value::from_scalar(0);
 pub struct TypeDescriptor {
     // Blob with candid-encoded type definitions.
     candid_data: Value,
-    // Blob with a list of `u32` offsets referring to the `candid_data`.
+    // Blob with a list of `usize` offsets referring to the `candid_data`.
```

Comment:
`u32` offsets would probably be enough, given 32-bit Candid encoding. BTW, will we ever compare type descriptors across 32 and 64 bit? I guess not, if the only migration path is via extended Candid, not EOP.

Reply:
Yes, agreed, this could be reduced to `u32` (put on my downgrade backlog for later).

It is not planned to compare type descriptors across 32 and 64 bit. Migration from 32-bit to 64-bit would require Candid stabilization. The type table is also stored for graph copying. So, changes to the type table would be difficult in the future (not impossible, but requires backwards support based on the persistence version number).